Add tricks file

LeandraH opened the pull request:
### Short description

It seems like a good idea to me to have a little file with pieces of knowledge and more or less cooking recipes that are special to our repo. I mostly keep these in a Notes file on my laptop but I guess it's better for the next person to put it somewhere publicly :) Maybe other people can also add things, now or when we come across something in the future.

ztefanie reviewed (approved):
Good idea, I also have a file like this, maybe you want to add some of these:

in web auf live cms wechseln: (command im browser console eingeben) window.localStorage.setItem('api-url', 'https://cms.integreat-app.de')

and i also have some debugging hints, but maybe they would need some more refinement/testing, as it is quite a while i used them, so feel free to just ignore it:

Debugging Native 1: 
    Strg M im Emulator für dev-menu und Inspector
    Im Terminal react-devtools ausführen, dann Strg M im Emulator
    adb logcat *:S ReactNative:V ReactNativeJS:V im Terminal um console.logs zu sehen
https://reactnative.dev/docs/debugging

Debugging Native 2:
Emulator starten, in Intellij "Start packager (React Symbol)" starten (Debug), dann öffnet browser debug fenster, reload app, app starten (ggf. Strg M und Start Debugging), breakpoints in intellij setzen und output in intellij

Debugging Web:
mit Intellij -> "Debug with Crome" im "JavaScript Debug" in Run Configs einrichten -> dann breakpoints in Intellij setzen -> läuft :)

im Browser: normal ausführen (grünes dreieck in INtellij) -> F12 -> Sources Tab -> da die selbe OrdnerStruktur wir in Intellij -> File öffnen -> Breakpoint setzen -> oben Symbole "step over usw."

ztefanie commented:
And maybe link the file to the main readme?

LeandraH commented:
Great idea! I added a link from the main Readme, and added the prod cms url but couldn't follow the debugging hints since they are very IntelliJ-heavy and I use VSCode, so I didn't feel comfortable describing them. But if you want to add them, feel free :)

steffenkleinle commented on docs/special-tricks.md:
```diff
@@ -0,0 +1,29 @@
+# Special tricks
```
How about:
```suggestion
# Development Tips and Tricks
```
I think the `specical` is kinda too unclear.

steffenkleinle commented on docs/special-tricks.md:
```diff
@@ -0,0 +1,29 @@
+This is a file in which we can collect some tricks that we might otherwise just put in private note files.
+We have a file to share these in instead here.
+
+### Changing the API in the web app
+
+In the browser console, run `window.localStorage.setItem('api-url', 'https://cms-test.integreat-app.de')`
+or whichever API (e.g. ''https://cms.integreat-app.de) you want to call. You can also do it manually in
+the local storage of the browser.
```
This is a duplicate (web/README.md#cms)

LeandraH replied:
It may be a duplicate but both Steffi and I had it in our notes so it's clearly not visible. I also don't really see this information changing so I don't think it's a problem that it's written down in two places.

steffenkleinle commented on docs/special-tricks.md:
```diff
@@ -0,0 +1,29 @@
+or whichever API (e.g. ''https://cms.integreat-app.de) you want to call. You can also do it manually in
+the local storage of the browser.
+
+### Triggering a deployment
+
+In `/tools`, use the following command: `yarn trigger-pipeline trigger <workflow from trigger-pipeline.ts>—api-token <CircleCI token>`.
```
This is a duplicate (docs/cicd.md#triggering-a-delivery)

steffenkleinle reviewed (approved):
I am not super happy about the duplicates, perhaps we could just link to the corresponding sections?
